cmd/tcl/testworkflow-toolkit/commands/services.go

```diff
@@ -159,6 +160,7 @@ func NewServicesCmd() *cobra.Command {
 						Pvcs: svcSpec.Pvcs,
 					}
 					spec.Steps[0].Run.ContainerConfig = testworkflowsv1.ContainerConfig{}
+					spec.Container.Env = testworkflowresolver.DedupeEnvVars(append(spec.Container.Env, config.Config().Execution.GlobalEnv...))
```

Comment:
Shouldn't it preserve order? In the current way, the global environments will have precedence over overrides.

Reply:
agree, it's better